### ISSUE TITLE
plotBrowserTrack() function returns .logError(f, fn = names(plotList)[i], info = "", errorList = NULL,:

### ISSUE THREAD

**Issue by mfisher1995** [GreenleafLab/ArchR#1156]
**Log File**
[ArchR-plotBrowserTrack-9b861458988c-Date-2021-11-12_Time-11-11-57.log](https://github.com/GreenleafLab/ArchR/files/7528925/ArchR-plotBrowserTrack-9b861458988c-Date-2021-11-12_Time-11-11-57.log)

**Issue with the plotBrowserTrack() function**

The plotBrowserTrack() function is not working, and it returning the following error:

Error in .logError(f, fn = names(plotList)[i], info = "", errorList = NULL,  : 
  Exiting See Error Above
In addition: Warning messages:
1: `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead. 
2: `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead. 
3: `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead. 
4: Removed 3 rows containing missing values (position_stack). 

I have already reinstalled the most up to date version of ArchR and tried installing various versions of ggplot2 in hopes of resolving this issue. So far I have been unable to resolve this on my own.

**Session Info**
> session_info()
─ Session info  ✔️  👶  🌁   ──────────────────────────────────────────────────────────────────────────────────────
 hash: check mark, baby, foggy

 setting  value
 version  R version 4.1.2 (2021-11-01)
 os       macOS Big Sur 10.16
 system   x86_64, darwin17.0
 ui       RStudio
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       America/New_York
 date     2021-11-12
 rstudio  1.3.1093 Apricot Nasturtium (desktop)
 pandoc   NA

**Comment by mfisher1995** [comment]
I was able to resolve the issue. In case anyone else has this issue in the future:

I recently updated my Mac to Monterey Version 12.0.1 which caused XQuartz to become the default plot viewer instead of the RStudio plotting pane. I beliebe this caused the issue, even if I was only trying to save the plot as a pdf and not view it. What resolved this was ultimately uninstalling and reinstalling Rstudio may be a Mac thing, a Monterey thing, not sure. Just wanted to share in case anyone else faced a similar problem.
